internal/core/src/segcore/SegmentSealedImpl.cpp

```diff
@@ -582,7 +582,7 @@ SegmentSealedImpl::search_ids(const BitsetView& bitset, Timestamp timestamp) con
             dst_offset.emplace_back(SegOffset(i));
         }
     }
-    return std::move(dst_offset);
+    return dst_offset;
```

Comment:
Side question, shall we remove the `std::move` call here as well? https://github.com/milvus-io/milvus/blob/804fbb6e27a56c0cacf8f6b4d0b2e173e3c4fe7e/internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp#L134

Reply:
Yes, I will remove it.